Makefile

```diff
@@ -200,59 +210,87 @@ verify-helm-lint:
 	@$(foreach chart,$(HELM_CHARTS),$(call lint-helm,$(chart)))
 .PHONY: verify-helm-lint
 
+ALL_K8S_TAG_FILES := $(shell \
+    find ./pkg -type f -name \*.go \
+    | xargs grep --color=never -l '^// *+k8s:' \
```

Comment:
that would match the generators as well (that keep this in a string template), I think I'd rather safely hard code a list

Reply:
I need to do this in my manager work as well, let's sync when you are back - maybe you'll be able to just pick up my work

Reply:
the go list (there can also be `find -printf` alternative) approach is live in https://github.com/scylladb/scylla-operator/pull/781

Reply:
picked up upstream changes